azure-swap.bash

```
@@ -0,0 +1,584 @@
+    
+    # Check if systemd is running
+    if ! systemctl is-system-running &>/dev/null && ! systemctl is-system-running --quiet 2>/dev/null; then
+        log_warn "systemd may not be fully operational"
+    fi
+}
+
+# Sanitize path inputs
+sanitize_path() {
+    local path="$1"
+    # Remove any dangerous characters and normalize
+    path="${path//[;<>|&\$`]/_}"
```

Shellcheck (0.11.0)

[error] Couldn't parse this backtick expansion. Fix to allow more checks.

(SC1073)